build.sbt

```diff
@@ -104,3 +104,20 @@ ThisBuild / githubWorkflowOSes := Seq("ubuntu-latest", "macos-latest", "windows-
 
 ThisBuild / githubWorkflowJavaVersions := Seq(
   JavaSpec.temurin("8"))
+
+ThisBuild / scalacOptions ++= List(
+  "-unchecked",
+  "-deprecation",
+  "-language:_",
+  "-encoding",
+  "UTF-8")
+
+ThisBuild / scalacOptions ++= {
+  if (insideCI.value) {
+    val log = sLog.value
```

Comment:
how about when a human needs to release this?

Reply:
Forgot about this, will be handled by our sbt plugin. Removing now

Reply:
@pjfanning Fixed